proof-producer/libs/output_artifacts/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ endif()
 target_link_libraries(proof_generatorOutputArtifacts
                       PUBLIC
                       crypto3::common
-                      Boost::log
```

Comment:
why log was removed?, as far i see it still used: https://github.com/NilFoundation/placeholder/blob/aaa4d94750a8ca3fe98f10ed6dd13c1d123055ea/proof-producer/libs/output_artifacts/include/nil/proof-generator/output_artifacts/assignment_table_writer.hpp#L20

Reply:
Fixed